docs/sources/tempo/operations/troubleshooting/metrics-generator.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Troubleshoot Metrics Generator
+menuTitle: Metrics Generator
+description: Gain an understanding of how to debug metrics quality issues.
+weight: 500
+---
+
+If you are concerned with data quality issues in the metrics generator, we'd first recommend:
+
+- Reviewing your telemetry pipeline to determine the number of dropped spans. We are only looking for major issues here.
+- Reviewing the [service graph documentation]({{< relref "../../metrics-generator/service_graphs" >}}) to understand how they are built.
+
+If everything seems ok from these two perspectives consider the following.
+
+## All metrics
+
+### Dropped spans in the distributor
+
+The distributor has a queue of outgoing spans to the metrics generators. If that queue is full then the distributor
+will drop spans before they reach the generator. Use the following metric to determine if that is happening:
+
+```
+sum(rate(tempo_distributor_queue_pushes_failures_total{}[1m]))
+```
+
+### Failed pushes to the generator
+
+For any number of reasons the distributor can fail a push to the generators. Use the following metric to
+determine if that is happening:
+
+```
+sum(rate(tempo_distributor_metrics_generator_pushes_failures_total{}[1m]))
+```
+
+### Discarded spans in the generator
+
+Spans are rejected from being considered by the metrics generator by a configurable slack time as well as due to user
+configurable filters. You can see the number of spans rejected by reason using this metric:
+
+```
+sum(rate(tempo_metrics_generator_spans_discarded_total{}[1m])) by (reason)
+```
+
+If you are dropping a lot of spans due to your filters, you will need to adjust them. If you are dropping a lot of spans
+due to the ingestion slack time, consider adjusting this setting:
+
+```
+metrics_generator:
+  metrics_ingestion_time_range_slack: 30s
+```
+
+### Max active series
+
+The generator protects itself and your remote write target by having a maximum number of series it will produce. To 
+determine if series are being dropped due to this limit check:
```

Comment:
Clarifying "it" in the first sentence. The antecedent for "it" is series, but I think we're referring to the metrics-generator.
```suggestion
The generator protects itself and your remote-write target by having a maximum number of series the generator produces.  
Use the `sum` below to determine if series are being dropped due to this limit:
```